src/OpenTK.OpenAL/Native/Extensions/Creative/EFX/Interfaces/IEFXSources.cs

```diff
@@ -33,8 +34,18 @@ internal interface IEFXSources
         /// <param name="source">The source.</param>
         /// <param name="param">The named property.</param>
         /// <param name="value">The value to set the property to.</param>
-        [NativeSymbol("Sourceb")]
-        void SetSourceProperty(uint source, EFXSourceBoolean param, bool value);
+        [NativeSymbol("Sourcei")]
+        void SetSourceProperty(uint source, EFXSourceBoolean param, [MarshalAs(UnmanagedType.I4)] bool value);
```

Comment:
Does bool marshalling work under Linux  and .Net Core as well. Because I could not get it to work in the past with DllImports(worked fine with mono). But I'm currently not sure anymore if it was just bool, or bools as parts of structs.
I'm probably gonna try it later on, after fixing my PR.

Reply:
Yes - we're not using DllImport at all, and ADL supports boolean marshalling very well.